app/models/framework/definition/RM3797.rb

```diff
@@ -6,6 +6,20 @@ class RM3797 < Base
 
       management_charge_rate BigDecimal('1')
 
+      PRODUCT_GROUPS = [
+        'Print Journal',
+        'Electronic Journal',
+        'Print and Electronic Journal'
+      ].freeze
+
+      UNIT_OF_MEASURE_VALUES = [
+        'Each'
+      ].freeze
+
+      UNSPSC_CODES = [
```

Comment:
This constant doesn't seem to be used in the validations..?

Reply:
Good catch! Forgot to add a `case_insensitive_inclusion`.